docs/pages/docs/handbook/_meta.json

```diff
@@ -0,0 +1,13 @@
+{
+  "index": "Introduction",
+  "package-installation": "Package Installation",
+  "workspaces": "Workspaces",
+  "migrating-to-a-monorepo": "Migrating to a Monorepo",
+  "dev": "Development Tasks",
+  "building-your-app": "Building Your App",
+  "sharing-code": "Sharing Code",
```

Comment:
Do you think sharing code should be broken up into section under workspaces or Migrating to a Monorepo.
It feels odd that this comes after Development tasks and Building your app.

I think it naturally wants to be tied to more info about organization before getting into tasks. 
 
<img width="703" alt="CleanShot 2022-09-14 at 10 38 41@2x" src="https://user-images.githubusercontent.com/5875536/190224596-2b187e14-1bdc-4fc3-bb98-a2ab62dfad19.png">

Reply:
My instinct is that most users who migrate to a monorepo want to:

FIRST set up their apps and make sure everything still works as expected
SECOND be able to share code between them.

So I think the current order makes sense.